src/main/java/org/apache/cassandra/distributed/api/IInstance.java

```diff
@@ -86,6 +86,8 @@ default int nodetool(String... commandAndArgs)
 
     void setMessagingVersion(InetSocketAddress addressAndPort, int version);
 
+    String getReleaseVersionString();
```

Comment:
as we spoke about, given that this is a breaking change all 4 branches must be updated (saying this more to be public than for you ^_^).